package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "s47_nkr_plu2_0",
+  "version": "1.0.0",
+  "description": "PROJECT TITLE",
```

Comment:
Consider adding a more descriptive message in the "description" field to provide more context about the project. [medium]